scripts/release/__tests__/label-patches.test.ts

```diff
@@ -138,6 +138,7 @@ it('should label the PR associated with cherry picks in the current branch', asy
             .trim()
         : text
     )
+    // eslint-disable-next-line @typescript-eslint/no-shadow
     .filter((it) => it !== '');
```

Comment:
it's best to change it so if we ever need to do a global find/replace from vitest's `it` to something else, it wouldn't affect this
```suggestion
    .filter((text) => text !== '');
```